request-management-api/request_api/services/documentservice.py

```diff
@@ -91,6 +92,14 @@ def copyrequestdocuments(self, ministryrequestid, documents, userid):
 
     def createministryrequestdocument(self, ministryrequestid, documentschema, userid):
         version = self.__getversionforrequest(ministryrequestid, "ministryrequest")
+        for document in documentschema['documents']:
+            if 'rrt' in document['category']:
+                #Create notification event for RRT document
+                message = f'RRT Uploaded on FOI Request {ministryrequestid}'
+                eventservice().attachmentevent(ministryrequestid, document, userid, "add", message)
+        #if 'rrt' in documentschema['documents']['category']:
+            #Create notification event for RRT document
+        #    eventservice().posteventforextension(ministryrequestid, '', userid, '' , "add")
         return FOIMinistryRequestDocument.createdocuments(ministryrequestid, version, documentschema['documents'], userid)
```

Comment:
keep the RRT related changes to events/attatment.py
Also, call the attachmentevent from the resource level(POST)